mina-sshd-api-core/pom.xml

```diff
@@ -49,6 +49,7 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>ssh-credentials</artifactId>
             <optional>true</optional>
+            <version>277.280.v1e86b_7d0056b_</version>
```

Comment:
My recommendation is to instead add this to the `dependencyManagement` section with a `TODO` comment requesting that the `dependency` entry be deleted once `bom` is used in a sufficiently new version. Then it is clear to anyone refreshing the plugin that there is something to be cleaned up.